subprojects/workers/src/main/java/org/gradle/workers/internal/ActionExecutionSpec.java

```diff
@@ -32,15 +32,15 @@
 public class ActionExecutionSpec implements WorkSpec {
     private final String displayName;
     private final Class<? extends Runnable> implementationClass;
-    private final File defaultDir;
-    private final File workingDir;
+    private final File idleWorkingDir;
```

Comment:
❌ Why would we send this with every request? Shouldn't this be known at worker startup?